packages/flutter/test/material/ink_paint_test.dart

```diff
@@ -9,6 +9,35 @@ import 'package:flutter_test/flutter_test.dart';
 import '../rendering/mock_canvas.dart';
 
 void main() {
+  testWidgets('The Ink widget renders a Container by default', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      Material(
+        child: Ink(),
+      ),
+    );
+    RenderBox container = tester.renderObject(find.byType(Container));
```

Comment:
`renderObject<RenderBox>` here and below however you can just use WidgeTester.getSize: 
```
expect(tester.getSize(find.byType(Container)), const Offset(800, 600));
```